engine/core/processors_sort.go

```diff
@@ -0,0 +1,80 @@
+package core
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/bmeg/grip/engine/logic"
+	"github.com/bmeg/grip/gdbi"
+	"github.com/bmeg/grip/gripql"
+	"github.com/bmeg/grip/log"
+)
+
+// Sort rows
+type Sort struct {
+	sortFields []*gripql.SortField
+}
+
+// FromBytes implements logic.SortConf.
+func (s *Sort) FromBytes(v []byte) gdbi.Traveler {
+	newTraveler := gdbi.BaseTraveler{}
+	err := json.Unmarshal(v, &newTraveler)
+	if err != nil {
+		log.Errorf("sort error: %s on %s", err, v)
```

Comment:
Testing this in grip-graphql this line is getting hit many times, but the sorting still appears to be working. Not sure why this is the case.

Reply:
Do you have a copy of the logged error?

Reply:
Query is:
{"level":"info","msg":"QUERY AFTER RENDER:  V().HasLabel(Observation).As(f0).Limit(100000).Sort([field:\"id\"]).Render({\"id\":\"$f0.id\"})","time":"2025-01-28T13:48:22-08:00"}

logs show many of:
{"level":"error","msg":"sort error: unexpected end of JSON input on {\"Current\":{\"ID\":\"001","time":"2025-01-28T13:44:29-08:00"}

with one of these in the middle and the last error log message:
{"level":"error","msg":"sort error: invalid character '|' looking for beginning of value on |","time":"2025-01-28T13:44:29-08:00"}